build.sbt

```diff
@@ -16,5 +16,6 @@ parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-  "com.novocode" % "junit-interface" % "0.11" % "test"
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "com.storm-enroute" %% "scalameter" % "0.8.2"
```

Comment:
I remember when I started using JMH I found some workloads on which JMH gave useful results and Scalameter didn't.  I don't remember any longer what this was, though, just that I specifically went looking for it based on the JMH documentation, and found it.  Thyme also got it wrong _but_ it usually got head-to-head comparisons qualitatively right due to explicitly testing them head-to-head.  (But Thyme is vulnerable to peculiarities of JVM history, since it runs entirely within the VM you've already got.)

Anyway, for now, I think either Scalameter or JMH is fine.  Scalameter is prettier.